## 59% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 20
Expected to fail tests: 1
Passed tests: 30
```
```
IDs of failed tests: 702840,702894,703086,727230,702855,702947,702958,702731,702844,702841,702733,704615,703629,702957,702843,702782,727229,703391,703503,703495 
```
```
IDs of expected to fail tests: 727232 
```
<h3>Failed tests (20)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=20463-unexpected-unsuported-collectibles&PR_ID=20657&APK_NAME=20657.apk&TR_CASE_IDS=702840,702894,703086,727230,702855,702947,702958,702731,702844,702841,702733,704615,703629,702957,702843,702782,727229,703391,703503,703495">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599896">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `communities-home-discover-card` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:261: in test_community_discovery
    self.home.community_card_item.wait_for_elements(seconds=120)
../views/base_element.py:138: in wait_for_elements
    raise TimeoutException(
 Device 1:  BaseElement by accessibility id:`community-card-item` is not found on the screen after wait_for_elements
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3c5bf00c462b462ca84e58d406149dbb?auth=55cb4d0bef157724deca3fdad4b5ce25#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599901">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:152: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:40: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599900">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `top-bar` </div><div>Device 1: Find `Button` by `accessibility id`: `top-bar` </div></blockquote></p>

```
critical/test_wallet.py:147: in test_wallet_send_eth
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.0 but expected to be -0.0001
E    Receiver balance is not updated on Etherscan, it is 0.0026 but expected to be 0.0027
E    Eth amount in the receivers wallet is 0.0026 but should be 0.0027
E    Eth amount in the senders wallet is 0.2001 but should be 0.2
E    The last transaction is not listed in activity for the receiver, expected timestamp is Today 7:51 PM
```

<br/><br/></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599879">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: SendMessageButton </div><div>Device 1: Looking for a message by text: AFTER (Edited) </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:400: in test_1_1_chat_edit_message
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 No edited message in 1-1 chat displayed
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e162d2912a8490c96c748b4ba7eb6d1?auth=4ce9b83807b5247592415eedf7996f64">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab7fe12d406a41fba51b7ad37f4336e2?auth=aabda76051776103fd91aa179e7eadd3#225">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599854">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//*[starts-with(@text,'Message 1')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 1: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:189: in test_1_1_chat_pin_messages
    self.chat_1.chat_element_by_text(self.message_1).wait_for_status_to_be("Delivered")
../views/chat_view.py:225: in wait_for_status_to_be
    raise TimeoutException("Message status was not changed to %s, it's %s" % (expected_status, current_status))
 Message status was not changed to Delivered, it's Sent
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e162d2912a8490c96c748b4ba7eb6d1?auth=4ce9b83807b5247592415eedf7996f64">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab7fe12d406a41fba51b7ad37f4336e2?auth=aabda76051776103fd91aa179e7eadd3#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599856">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599861">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><p><blockquote><div>Device 1: Long press on `ChatElementByText` until expected element is shown </div><div>Device 1: Find `Button` by `xpath`: `//*[@text="Reply"]` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:129: in test_1_1_chat_emoji_send_reply_and_open_link
    self.chat_1.quote_message(emoji_unicode)
../views/chat_view.py:1041: in quote_message
    self.reply_message_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@text="Reply"]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e162d2912a8490c96c748b4ba7eb6d1?auth=4ce9b83807b5247592415eedf7996f64">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab7fe12d406a41fba51b7ad37f4336e2?auth=aabda76051776103fd91aa179e7eadd3#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599893">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599894">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The test session has already finished, and can't receive further commands. For help please check https://docs.saucelabs.com/dev/error-messages 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599883">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `LogInButton` by `accessibility id`: `login-button` </div><div>Device 2: Tap on found: LogInButton </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:274: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bb5d583da5f742d3b9d2edaaeb71f293?auth=e4a52654ce462b2e37936e3e3f0f9182">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/08c44f745cb54af08206d4acb9d9e353?auth=e05bc0c6594b44ab0943c89dde5be13b">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599871">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:532: in test_community_emoji_send_copy_paste_reply
    self.channel_1.navigate_back_to_chat_view()
../views/base_view.py:443: in navigate_back_to_chat_view
    while not element.is_element_displayed(1) and counter <= attempts:
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#302">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599882">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:672: in test_community_contact_block_unblock_offline
    if not channel.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#311">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599887">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599874">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:602: in test_community_links_with_previews_github_youtube_twitter_gif_send_enable
    if not home.chat_floating_screen.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#305">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599872">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:647: in test_community_unread_messages_badge
    self.channel_1.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#308">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599899">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599873">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for a message by text: Message AFTER edit 2 (Edited) </div><div>Device 2: Find `ChatElementByText` by `xpath`: `//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:378: in test_community_message_edit
    self.channel_2.set_reaction(message_text_after_edit)
../views/chat_view.py:1052: in set_reaction
    self.chat_element_by_text(message).long_press_until_element_is_shown(element)
../views/base_element.py:327: in long_press_until_element_is_shown
    element = self.find_element()
../views/chat_view.py:116: in find_element
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: ChatElementByText by xpath:`//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599886">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599885">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `LogInButton` by `accessibility id`: `login-button` </div><div>Device 2: Tap on found: LogInButton </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:425: in join_community
    self.community_status_joined.wait_for_visibility_of_element(60)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by accessibility id:`status-tag-positive` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8aee246b07dc493083fbb13bfa9e4653?auth=a34d0fb272f426f94264c267ed93ca43">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/77abaf9e6abb4ecaa904cb79da590349?auth=100b1e940cb927db99eecb7a734fc557">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599897">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
critical/chats/test_public_chat_browsing.py:1127: in test_community_join_when_node_owner_offline
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/12c5fc3d52cf40eaab2d3bfdd15cfb34?auth=df67b8d84c22a40e4a04d4bf2c203f08#506">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1594cdb1054b4eddb1ad2a46bf1196b0?auth=2071b33fb48321961e1433dfdff987d5#473">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599903">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EditBox` by `accessibility id`: `add-address-to-watch` </div><div>Device 1: Type `0x8d2413447ff297d30bdc475f6d5cb00254685aae` to `EditBox` </div></blockquote></p>

```
critical/test_wallet.py:237: in test_wallet_add_remove_watch_only_account
    self.wallet_view.add_watch_only_account(address=address_to_watch, account_name=new_account_name)
../views/wallet_view.py:150: in add_watch_only_account
    self.account_has_activity_label.wait_for_visibility_of_element()
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: Text by accessibility id:`account-has-activity` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/718bbe8e47da4100af7ca6a43c053e16?auth=9d31857c4ed9191bf0c8694d83ec77c2#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (30)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599857">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3c5bf00c462b462ca84e58d406149dbb?auth=55cb4d0bef157724deca3fdad4b5ce25#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599876">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3c5bf00c462b462ca84e58d406149dbb?auth=55cb4d0bef157724deca3fdad4b5ce25#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599888">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3c5bf00c462b462ca84e58d406149dbb?auth=55cb4d0bef157724deca3fdad4b5ce25#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599881">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3c5bf00c462b462ca84e58d406149dbb?auth=55cb4d0bef157724deca3fdad4b5ce25#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599892">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3c5bf00c462b462ca84e58d406149dbb?auth=55cb4d0bef157724deca3fdad4b5ce25#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599902">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/718bbe8e47da4100af7ca6a43c053e16?auth=9d31857c4ed9191bf0c8694d83ec77c2#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599895">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/59631bcf5e4a4a44abca4a4c8f360d4a?auth=3baa81cabc30862d4384a9a653d46c3b#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ccc4ba5123ee4a88abe0f1af08aa4d24?auth=fbda5ef3902e0fb513fe3a262e03dd18#112">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599862">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/59631bcf5e4a4a44abca4a4c8f360d4a?auth=3baa81cabc30862d4384a9a653d46c3b#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ccc4ba5123ee4a88abe0f1af08aa4d24?auth=fbda5ef3902e0fb513fe3a262e03dd18#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599863">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/59631bcf5e4a4a44abca4a4c8f360d4a?auth=3baa81cabc30862d4384a9a653d46c3b#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ccc4ba5123ee4a88abe0f1af08aa4d24?auth=fbda5ef3902e0fb513fe3a262e03dd18#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599859">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6a01aa141894422fa165f3d16915de0a?auth=3f8b6900c17414df5f0a4f75adb89883#219">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599898">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6a01aa141894422fa165f3d16915de0a?auth=3f8b6900c17414df5f0a4f75adb89883#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599853">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e162d2912a8490c96c748b4ba7eb6d1?auth=4ce9b83807b5247592415eedf7996f64">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab7fe12d406a41fba51b7ad37f4336e2?auth=aabda76051776103fd91aa179e7eadd3#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599858">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e162d2912a8490c96c748b4ba7eb6d1?auth=4ce9b83807b5247592415eedf7996f64">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab7fe12d406a41fba51b7ad37f4336e2?auth=aabda76051776103fd91aa179e7eadd3#144">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599868">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0e162d2912a8490c96c748b4ba7eb6d1?auth=4ce9b83807b5247592415eedf7996f64">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ab7fe12d406a41fba51b7ad37f4336e2?auth=aabda76051776103fd91aa179e7eadd3#194">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599878">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a1f229f3e4c448dba6280fd0ddfbc88?auth=6e4223938db9bae68c0b292e0e5f7adb#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c811236cebfd4ba78a8d8c8e460026a8?auth=c3cd5885a8157edbacfad71d10bd9832#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599877">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a1f229f3e4c448dba6280fd0ddfbc88?auth=6e4223938db9bae68c0b292e0e5f7adb#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c811236cebfd4ba78a8d8c8e460026a8?auth=c3cd5885a8157edbacfad71d10bd9832#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599860">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a1f229f3e4c448dba6280fd0ddfbc88?auth=6e4223938db9bae68c0b292e0e5f7adb#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c811236cebfd4ba78a8d8c8e460026a8?auth=c3cd5885a8157edbacfad71d10bd9832#210">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599890">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/91bb0f039c754de594a1b0d1886f89a8?auth=4f30e162c4d9f4b51ec790a1bcbcc533#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/96d0180de74a4511925c295cf5bd424f?auth=efc69b72b50a15eb6ba62080fa35e556#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a715a98da4f4898b0bb5f8f43263aaf?auth=68ccea658b409cd8f23e3652f734079f#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599865">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/91bb0f039c754de594a1b0d1886f89a8?auth=4f30e162c4d9f4b51ec790a1bcbcc533#156">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/96d0180de74a4511925c295cf5bd424f?auth=efc69b72b50a15eb6ba62080fa35e556#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a715a98da4f4898b0bb5f8f43263aaf?auth=68ccea658b409cd8f23e3652f734079f#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599866">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/91bb0f039c754de594a1b0d1886f89a8?auth=4f30e162c4d9f4b51ec790a1bcbcc533#320">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/96d0180de74a4511925c295cf5bd424f?auth=efc69b72b50a15eb6ba62080fa35e556#203">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a715a98da4f4898b0bb5f8f43263aaf?auth=68ccea658b409cd8f23e3652f734079f#204">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599855">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/91bb0f039c754de594a1b0d1886f89a8?auth=4f30e162c4d9f4b51ec790a1bcbcc533#353">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/96d0180de74a4511925c295cf5bd424f?auth=efc69b72b50a15eb6ba62080fa35e556#230">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a715a98da4f4898b0bb5f8f43263aaf?auth=68ccea658b409cd8f23e3652f734079f#231">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599891">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/91bb0f039c754de594a1b0d1886f89a8?auth=4f30e162c4d9f4b51ec790a1bcbcc533#280">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/96d0180de74a4511925c295cf5bd424f?auth=efc69b72b50a15eb6ba62080fa35e556#168">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2a715a98da4f4898b0bb5f8f43263aaf?auth=68ccea658b409cd8f23e3652f734079f#142">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599870">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#198">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599869">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599880">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#254">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599889">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6d770c2c67364e4a90b80a742c1160e4?auth=54e3bec01bb5c9351a8030a1332c382b#229">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f320dfb493d04c8fb4153b6dcf4f1023?auth=61140740c9239183dfb88dc79cfddec6">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599875">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/12c5fc3d52cf40eaab2d3bfdd15cfb34?auth=df67b8d84c22a40e4a04d4bf2c203f08#301">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1594cdb1054b4eddb1ad2a46bf1196b0?auth=2071b33fb48321961e1433dfdff987d5#270">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599864">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/12c5fc3d52cf40eaab2d3bfdd15cfb34?auth=df67b8d84c22a40e4a04d4bf2c203f08#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1594cdb1054b4eddb1ad2a46bf1196b0?auth=2071b33fb48321961e1433dfdff987d5#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599867">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/12c5fc3d52cf40eaab2d3bfdd15cfb34?auth=df67b8d84c22a40e4a04d4bf2c203f08#185">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1594cdb1054b4eddb1ad2a46bf1196b0?auth=2071b33fb48321961e1433dfdff987d5#168">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3599884">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/12c5fc3d52cf40eaab2d3bfdd15cfb34?auth=df67b8d84c22a40e4a04d4bf2c203f08#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1594cdb1054b4eddb1ad2a46bf1196b0?auth=2071b33fb48321961e1433dfdff987d5#292">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/quo/components/profile/collectible_list_item/style.cljs
+++ b/src/quo/components/profile/collectible_list_item/style.cljs
@@ -11,7 +11,7 @@
 (defn fallback
   [{:keys [theme opacity]}]
   [{:opacity opacity}
-   {:opacity          default-opacity-for-image
+   {:opacity          1
     :background-color (colors/theme-colors colors/neutral-2_5 colors/neutral-90 theme)
     :border-style     :dashed
     :border-color     (colors/theme-colors colors/neutral-20 colors/neutral-80 theme)
@@ -60,8 +60,13 @@
    :padding-right  0
    :padding-top    4})
 
-(def card-detail-text
-  {:flex 1})
+(defn card-detail-text
+  [empty-name? theme]
+  {:flex         1
+   :margin-right 8
+   :color        (if empty-name?
+                   (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
+                   (colors/theme-colors colors/neutral-100 colors/white theme))})
 
 (defn card-loader
   [opacity]
@@ -115,6 +120,7 @@
   [{:opacity opacity}
    {:flex           1
     :flex-direction :row
+    :column-gap     8
     :opacity        default-opacity-for-image}])
 
 (defn supported-file

--- a/src/quo/components/profile/collectible_list_item/view.cljs
+++ b/src/quo/components/profile/collectible_list_item/view.cljs
@@ -1,5 +1,6 @@
 (ns quo.components.profile.collectible-list-item.view
   (:require
+    [clojure.string :as string]
     [quo.components.avatars.collection-avatar.view :as collection-avatar]
     [quo.components.counter.collectible-counter.view :as collectible-counter]
     [quo.components.icon :as icon]
@@ -21,7 +22,7 @@
 (def cached-load-time 200)
 (def error-wait-time 800)
 
-(defn on-load-end
+(defn animate-loading-image
   [{:keys [load-time set-state loader-opacity image-opacity]}]
   (reanimated/animate loader-opacity 0 timing-options-out)
   (reanimated/animate image-opacity 1 timing-options-in)
@@ -60,12 +61,12 @@
                  (assoc prev-state :avatar-loaded? true)))))
 
 (defn- fallback-view
-  [{:keys [label theme image-opacity]}]
-  [reanimated/view
-   {:style (style/fallback {:opacity image-opacity
-                            :theme   theme})}
+  [{:keys [label theme image-opacity on-load-end]}]
+  (rn/use-mount on-load-end)
+  [reanimated/view {:style (style/fallback {:opacity image-opacity :theme theme})}
    [rn/view
-    [icon/icon :i/sad {:color (colors/theme-colors colors/neutral-40 colors/neutral-50 theme)}]]
+    [icon/icon :i/sad
+     {:color (colors/theme-colors colors/neutral-40 colors/neutral-50 theme)}]]
    [rn/view {:style {:height 4}}]
    [text/text
     {:size  :paragraph-2
@@ -93,31 +94,43 @@
   [{:keys [community? avatar-image-src collectible-name theme state set-state]}]
   (let [loader-opacity            (reanimated/use-shared-value 1)
         avatar-opacity            (reanimated/use-shared-value 0)
-        [load-time set-load-time] (rn/use-state (datetime/now))]
+        [load-time set-load-time] (rn/use-state (datetime/now))
+        set-avatar-loaded         (fn []
+                                    (on-load-avatar {:set-state      set-state
+                                                     :load-time      load-time
+                                                     :loader-opacity loader-opacity
+                                                     :avatar-opacity avatar-opacity}))
+        empty-name?               (string/blank? collectible-name)]
+    (rn/use-mount (fn []
+                    (when (string/blank? avatar-image-src)
+                      (set-avatar-loaded))))
     [rn/view {:style style/card-details-container}
      [reanimated/view {:style (style/avatar-container avatar-opacity)}
-      (if community?
-        [preview-list/view
-         {:type :communities
-          :size :size-20}
+      (cond
+        (string/blank? avatar-image-src)
+        [loading-square theme]
+
+        community?
+        [preview-list/view {:type :communities :size :size-20}
          [avatar-image-src]]
+
+        :else
         [collection-avatar/view
          {:size        :size-20
           :on-start    #(set-load-time (fn [start-time] (- (datetime/now) start-time)))
-          :on-load-end #(on-load-avatar {:set-state      set-state
-                                         :load-time      load-time
-                                         :loader-opacity loader-opacity
-                                         :avatar-opacity avatar-opacity})
+          :on-load-end set-avatar-loaded
           :image       avatar-image-src}])
-      [rn/view {:style {:width 8}}]
       [text/text
        {:size            :paragraph-1
         :weight          :semi-bold
         :ellipsize-mode  :tail
         :number-of-lines 1
-        :style           style/card-detail-text}
-       collectible-name]]
-     (when (not (:avatar-loaded? state))
+        :style           (style/card-detail-text empty-name? theme)}
+       (if empty-name?
+         (i18n/label :t/unknown)
+         collectible-name)]]
+
+     (when-not (:avatar-loaded? state)
        [reanimated/view {:style (style/card-loader loader-opacity)}
         [loading-square theme]
         [loading-message theme]])]))
@@ -128,40 +141,44 @@
   (let [theme                     (quo.theme/use-theme)
         loader-opacity            (reanimated/use-shared-value (if supported-file? 1 0))
         image-opacity             (reanimated/use-shared-value (if supported-file? 0 1))
-        [load-time set-load-time] (rn/use-state (datetime/now))]
+        [load-time set-load-time] (rn/use-state (datetime/now))
+        set-image-loaded          (fn []
+                                    (animate-loading-image {:load-time      load-time
+                                                            :set-state      set-state
+                                                            :loader-opacity loader-opacity
+                                                            :image-opacity  image-opacity}))]
     [rn/view {:style (style/card-view-container theme)}
      [rn/view {:style {:aspect-ratio 1}}
       (cond
-        (:image-error? state)
-        [fallback-view
-         {:image-opacity image-opacity
-          :theme         theme
-          :label         (i18n/label :t/cant-fetch-info)}]
+        (:image-error? state)        [fallback-view
+                                      {:image-opacity image-opacity
+                                       :on-load-end   set-image-loaded
+                                       :theme         theme
+                                       :label         (i18n/label :t/cant-fetch-info)}]
+        (not supported-file?)        [fallback-view
+                                      {:image-opacity image-opacity
+                                       :on-load-end   set-image-loaded
+                                       :theme         theme
+                                       :label         (i18n/label :t/unsupported-file)}]
+        (not (:image-loaded? state)) [loading-image
+                                      {:loader-opacity       loader-opacity
+                                       :theme                theme
+                                       :gradient-color-index gradient-color-index}])
 
-        (not supported-file?)
-        [fallback-view
-         {:image-opacity image-opacity
-          :theme         theme
-          :label         (i18n/label :t/unsupported-file)}]
-
-        (not (:image-loaded? state))
-        [loading-image
-         {:loader-opacity       loader-opacity
-          :theme                theme
-          :gradient-color-index gradient-color-index}])
       (when supported-file?
         [reanimated/view {:style (style/supported-file image-opacity)}
          [rn/image
           {:style         style/image
            :on-load       #(on-load % set-state)
            :on-load-start #(set-load-time (fn [start-time] (- (datetime/now) start-time)))
-           :on-load-end   #(on-load-end {:load-time      load-time
-                                         :set-state      set-state
-                                         :loader-opacity loader-opacity
-                                         :image-opacity  image-opacity})
+           :on-load-end   set-image-loaded
            :on-error      #(on-load-error set-state)
            :source        image-src}]])]
-     (when (and (:image-loaded? state) (not (:image-error? state)) counter)
+
+     (when (and (or (:image-loaded? state)
+                    (not supported-file?))
+                (not (:image-error? state))
+                counter)
        [collectible-counter/view
         {:container-style style/collectible-counter
          :size            :size-24
@@ -206,10 +223,10 @@
          {:style         style/image
           :on-load       #(on-load % set-state)
           :on-load-start #(set-load-time (fn [start-time] (- (datetime/now) start-time)))
-          :on-load-end   #(on-load-end {:load-time      load-time
-                                        :set-state      set-state
-                                        :loader-opacity loader-opacity
-                                        :image-opacity  image-opacity})
+          :on-load-end   #(animate-loading-image {:load-time      load-time
+                                                  :set-state      set-state
+                                                  :loader-opacity loader-opacity
+                                                  :image-opacity  image-opacity})
           :on-error      #(on-load-error set-state)
           :source        image-src}]])
      (when (and (:image-loaded? state) (not (:image-error? state)) counter)

--- a/src/status_im/subs/wallet/collectibles.cljs
+++ b/src/status_im/subs/wallet/collectibles.cljs
@@ -116,10 +116,11 @@
    (get-in collectible [:collectible-data :traits])))
 
 (re-frame/reg-sub
- :wallet/last-collectible-details-owner
- :<- [:wallet/last-collectible-details]
- :<- [:wallet]
- (fn [[collectible wallet]]
-   (let [address (:address (first (:ownership collectible)))
-         account (get-in wallet [:accounts address])]
-     account)))
+ :wallet/collectible-details-owner
+ :<- [:wallet/accounts]
+ (fn [accounts [_ collectible]]
+   (let [collectible-address (-> collectible :ownership first :address)]
+     (some #(when (= (:address %) collectible-address)
+              %)
+           accounts))))
+

